### DISCUSSION TITLE
use ExpoSDK\Expo; Error.

### DISCUSSION THREAD

**Discussion by SoloAppLab** [ctwillie/expo-server-sdk-php#35] [Q&A] [answered]
Ok, I know I'm missing something super easy. But I'm fairly new to PHP. I've installed the package using composer and everything installed fine. But when I go to send a notification it says use ExpoSDK\Expo; not found.

What am I missing here? This is literally the last step for my project to be completed. :) Thanks in advance to anyone who can help me out!

**Comment by ctwillie** [comment, marked as answer]
@SoloAppLab Hey! I'm assuming you are using PSR-4 autoloading. Below is an example to get you going. Be sure you are pulling in your autoload file (require/require_once).

```php
<?php

use ExpoSDK\Expo;
use ExpoSDK\ExpoMessage;

// Require your autoload file.. Change the path to where it's actually located
require_once __DIR__ . '/vendor/autoload.php';


$expo = new Expo();

$messages = [
    new ExpoMessage([
        'title' => 'Test message',
        'body' => 'This is a test message',
    ]),
];

$recipients = [
    'ExponentPushToken[xxxx-xxxx-xxxx]',
    'ExponentPushToken[zzzz-zzzz-zzzz]'
];

$response = $expo->send($messages)->to($recipients)->push();
var_dump($response->getData());
```

**Reply by SoloAppLab** [reply]
@ctwillie that did it! Thank you! 🙏 I knew it was something dumb on my end. 🤦‍♂️😅.

**Reply by ctwillie** [reply]
Awesome. Glad that resolved it!
